clusters.json

```diff
@@ -0,0 +1,41 @@
+{
+    "uan*": {
+        "name": "LUMI",
+        "vars": {
+            "HF_HOME": "/pfs/lustrep4/projappl/project_462000963/evals/hf_data",
+            "OUTPUT_DIR": "/pfs/lustrep4/projappl/project_462000963/evals/${USER}",
```

Comment:
Should likely be `EVAL_OUTPUT_DIR` or anything more specific.
Can we use the project name as env variable rather than hardcoding it?